colin/checks/labels/architecture.py

```diff
@@ -0,0 +1,16 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class ArchitectureLabelCheck(LabelCheck):
+
+    def __init__(self):
+        super().__init__(name="architecture_label",
+                         message="Label 'architecture' has to be specified.",
+                         description="Architecture the software in the image should target "
+                                     "(Optional: if omitted, it will be built "
+                                     "for all supported Fedora Architectures)",
+                         reference_url="https://fedoraproject.org/wiki/Container:Guidelines#LABELS",
+                         tags=["architecture", "label", "required"],
```

Comment:
Architecture label is optional, based on the our document.

Reply:
I'm not sure, what to add here, because fo `RHEL` it required, but optional for Fedora. Both of them?

Reply:
I would prefer it if possible.